pkg/apis/meta.go

```diff
@@ -0,0 +1,303 @@
+package apis
```

Comment:
I'm not sure this is the best final location, but for sure in this part of the tree.

Everything in this tree should be apache2 and easy to import into 3rd party clients

Reply:
open to suggestions, the constants at least will be something people will want to import but possibly it should be in a subpackage